integration/tests/iam_v1_to_v2_force_upgrade_diagnostics.sh

```diff
@@ -43,3 +43,11 @@ do_deploy() {
         --skip-preflight \
         --debug
 }
+
+do_prepare_upgrade() {
+  # use latest current here
+  # shellcheck disable=SC2154
+  download_manifest_version "current" "20220329091442" "$test_manifest_dir/20220329091442.json"
```

Comment:
As this code is same in some other files, can we write a common function and call inside do_prepare_upgrade function

Reply:
done